tests/int_test.ml

```diff
@@ -0,0 +1,28 @@
+open InfiniteJest
+open InfiniteJest.Test
+open Int
+
+let suite = 
+    describe "Int" (fun () -> [
+        test "pow" (fun () -> 
+            pow 2 1 |> Expect.toEqual 2
+        );
+        test "pow - raise" (fun () ->
+            (fun () -> pow 0 0) |> Expect.toRaise
+        );
+        test "pow - negative" (fun () -> 
+            (fun () -> pow 1 (~-1)) |> Expect.toRaise
+        );
```

Comment:
I like to group the tests for the same functions by adding a blank line to separate them from the tests of other functions.